src/ibl_to_nwb/ecephys/convert_session.py

```diff
@@ -0,0 +1,20 @@
+from src.ibl_to_nwb.tools.ecephys.spikeglx_to_nwb import write_recording
+
+# This is a unique identifier for an experimental session, required for streaming the data from Alyx database
+pid = "da8dfec1-d265-44e8-84ce-6ae9c109b8bd"
```

Comment:
From https://int-brain-lab.github.io/iblenv/notebooks_external/loading_raw_ephys_data.html#Option-2:-Download-all-of-raw-ephys-data, are we sure that the `pid` is the experimental identifier? I had thought that `eid` was for `experimental_id` and `pid` was for `probe_id` (could be interpreted as a single probe for a single session in a single experiment, but that experiment may use other probes as well)

Reply:
Ah no, you're right. I'll have to fix that.